content/showcase/_index.md

```diff
@@ -5,72 +5,80 @@ url_use_grpc: https://www.cncf.io/case-studies/?_sft_lf-project=grpc
 spelling: cSpell:ignore youtube Kubernetes Nulab Protop
 ---
 
+{{< blocks/cover height="sm" color="primary" >}}
+{{< page/header >}}
+{{< /blocks/cover >}}
+
+<div class="container">
+<div class="row my-5">
+
+<div class="d-lg-none col-9">
+<div class="td-toc td-toc--inline">
+{{< page/toc collapsed=true >}}
+</div>
+</div>
```

Comment:
Can some of this be put into a shortcode (and used in the other pages that include these elements)?

Reply:
I've consolidated this a bit - while shortcodes can be nested in the same template, they can't be nested in each other - the `div`s wrapping the `page/toc` concerned with layout have been added to that shortcode along with a `placement` param.